public/static/docs/api-reference/index.md

```diff
@@ -0,0 +1,30 @@
+# Python API
```

Comment:
could we check other available APIs references - what do the include in sections like this?

Reply:
I think typically they're just separated either
- by features/use cases e.g. [numpy](https://docs.scipy.org/doc/numpy/reference/) or [pandas](https://pandas.pydata.org/pandas-docs/stable/reference/index.html) (BTW includes an "Extensions" section re `dvcx`) – but these is best for huge libraries
- hierarchically by module->property/class/function e.g. https://developers.arcgis.com/python/api-reference/ – which is kind of what we do with each function, but they're just not listed in the index in our case.
- chronologically or as a story/guide e.g. https://github.com/sidecars/python-quickbooks or other SDKs, which is what we're going for in the index page.